zebedee-content-tools/publishing-timings/README.md

```diff
@@ -0,0 +1,50 @@
+# Publishing timings tool
+
+## Overview
+
+This tool traverses the publish-log directory in the zebedee content workspace and extracts timings to provide an
+illustration of the length of time scheduled collections and the releases they are part of take to pre-publish and
+publish. It does this by identifying the relevant collections, getting the earliest and latest timestamps in the file
+related to [pre-]publishing activities. It then augments this with file counts and
```

Comment:
Is there something missing after the 'and' ...

Reply:
Yes. The rest of the paragraph. 🤦‍♀️ Thanks for spotting.